omnibus/config/software/chefdk.rb

```diff
@@ -39,13 +39,9 @@
 
 relative_path "chef-dk"
 
-if windows?
-  dependency "ruby-windows"
-  dependency "ruby-windows-devkit"
-else
-  dependency "libffi" if debian?
-  dependency "ruby"
-end
+dependency "libffi" if debian?
```

Comment:
Don't need to specify ffi any more.  It comes with ruby.  We used to need it if we were using chef-gem instead of chef.